back/app/serializers/web_api/v1/permission_serializer.rb

```diff
@@ -11,6 +13,67 @@ class WebApi::V1::PermissionSerializer < WebApi::V1::BaseSerializer
     object.everyone_tracking_enabled?
   end
 
+  attribute :user_fields_in_form do |permission|
+    phase = permission.permission_scope
+
+    unless phase.is_a?(Phase)
+      next false
+    end
```

Comment:
Maybe it's personal preference, but I tend favour one line conditionals when using unless and there is only one line in the output of the conditional:
```suggestion
    next false unless phase.is_a?(Phase)
```